### ISSUE TITLE
Integrate flake8 into build/CI pipeline

### ISSUE THREAD

**Issue by cclauss** [localstack/localstack#291]
`fail()`is an undefined name in https://github.com/localstack/localstack/blob/master/tests/integration/test_cloudformation.py#L79  Where is `fail()`defined?  Perhaps I am just missing something.  This line could be:
```python
assert False, "Should raise ValidationError"
```

flake8 testing of https://github.com/localstack/localstack on Python 3.6.2

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./tests/integration/test_cloudformation.py:79:9: F821 undefined name 'fail'
        fail("Should raise ValidationError")
        ^
```

**Comment by whummer** [comment]
Thanks for reporting @cclauss . This is fixed in #295 . I'll keep this issue open to remind us to integrate `flake8` into the build pipeline.

**Comment by whummer** [comment]
Fixed in #299
